crates/compiler/load_internal/src/work.rs

```diff
@@ -484,8 +486,9 @@ impl<'a> Dependencies<'a> {
                 self.add_dependency(module_dep, module, Phase::MakeSpecializations);
                 self.add_dependency(ModuleId::DERIVED_GEN, module, Phase::MakeSpecializations);
 
-                // `module_dep` can't make its specializations until the current module does.
-                info.has_pred = true;
```

Comment:
Is this the only really change. Crazy that this is the fix. Also, I guess skipping env.roc. Just amazed this is such a small diff.

Reply:
Yeah, the only delta between roc dev and roc build aside runtime is how the build graph is created. So it would make sense we "forgot" this